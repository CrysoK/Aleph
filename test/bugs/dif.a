// Diferencia de conjuntos con resultado incorrecto.

let a = {[10,20],4,5,6,4,['a','b',{'c'}],5,4,3,1,'x'}
let b = {[10,20],7,8,7,{'c',{},['a']},9,6,4,3,2}

println(a)
println(b)
println()

println(a + b)
println(a * b)
println(a - b) // BUG AQU√ç
println(b - a)
println(a - b)
