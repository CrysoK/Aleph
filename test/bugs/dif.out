{1, 3, 4, 5, 6, 'x', [10, 20], ['a', 'b', {'c'}]}
{2, 3, 4, 6, 7, 8, 9, {'c', {}, ['a']}, [10, 20]}

{1, 2, 3, 4, 5, 6, 7, 8, 9, 'x', {'c', {}, ['a']}, [10, 20], ['a', 'b', {'c'}]}
{3, 4, 6, [10, 20]}
{1, 5, 'x', ['a', 'b', {'c'}]}
{2, 7, 8, 9, {'c', {}, ['a']}}
{1, 5, 'x', ['a', 'b', {'c'}]}
